units/systemd-logind-varlink.socket

```diff
@@ -0,0 +1,18 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=User Login Management Varlink Socket
+Documentation=man:systemd-logind.service(8)
+
+[Socket]
+ListenStream=/run/systemd/io.systemd.Login
+FileDescriptorName=varlink
+SocketMode=0666
+Service=systemd-logind.service
```

Comment:
Is it not necessary to add Socket=systemd-logind-varlink.socket in the service?

Reply:
See https://github.com/systemd/systemd/pull/37309/commits/e803ec1e253a2d6593c3a7cc075f6f0bc4e696ac